<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="GameplayWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <widget class="QLabel" name="PlayerLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[PLAYER NAME]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="LansButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>190</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Lans</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RobotaButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>190</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Do roboty</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HaraczButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>310</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Haracz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BMWButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>310</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>BMW</string>
     </property>
    </widget>
    <widget class="QPushButton" name="IvanButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>450</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Ivan</string>
     </property>
    </widget>
    <widget class="QLabel" name="SzacunLabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>90</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="KasaLabel">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>90</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="BMLabel">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>90</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="HaraczLabel">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>90</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="GameResultWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <widget class="QLabel" name="ResultLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>1021</width>
       <height>361</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>[Result]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="PlayerCreationWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="PlayerNameEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nazwa gracza</string>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="isRobotCheckbox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>260</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Gracz SI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPlayerButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>340</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Dodaj gracza</string>
     </property>
    </widget>
    <widget class="QPushButton" name="endCreationButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>440</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Zako≈Ñcz</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
